deepchem/models/losses.py

```diff
@@ -465,6 +470,216 @@ def loss(inputs):
         return loss
 
 
+class MolGraphConvMILoss():
+    """
+    Losses used to maximize the mutual information between molecular graph encodings.
+    Encodings can be generated from molecular graphs using a graph convolutional network, and the mutual information is calculated between subgraphs (local encodings) and the entire molecule (global encodings) in order to enable unsupervised training.
+
+    This class supports only PyTorch.
+
+    Parameters:
+    device: torch.device
+        Device to run the model on.
+    measure: str
+        The divergence measure to use for the unsupervised loss. Options are 'GAN', 'JSD',
+        'KL', 'RKL', 'X2', 'DV', 'H2', or 'W1'.
+    average_loss: bool
+        Whether to average the loss over the batch
+    """
+
+    def __init__(self, device, measure='JSD', average_loss=True):
+        self.device = device
+        self.measure = measure
+        self.average_loss = average_loss
+
+    def local_global_loss(self, l_enc, g_enc, index):
```

Comment:
This class isn't following the convention the other losses follow. This function should be called `_create_pytorch_loss`. See https://github.com/deepchem/deepchem/blob/master/deepchem/models/losses.py#L208 for a representative example